cmd/build.go

```diff
@@ -292,18 +228,35 @@ func (c buildConfig) Prompt() (buildConfig, error) {
 				Message: "Project path:",
 				Default: c.Path,
 			},
-			Validate: survey.Required,
```

Comment:
Best to concentrate validation logic in one place as much as possible, and the client API validates and issues the final error (if any)